### PR TITLE
added optional recordcount and operationcount args to dataserving cli…

### PR DESCRIPTION
**Pull request by ustiugov**
…ent; fixed the instructions on the website

### CODE REVIEW

File: benchmarks/data-serving/client/docker-entrypoint.sh [hunk 1]
```diff
@@ -3,6 +3,14 @@
...
+if [ ! -z "$RECORDCOUNT" ]; then
+    RECORDCOUNT="-p recordcount=$RECORDCOUNT"
+fi
 
-/ycsb/bin/ycsb run cassandra-10 -p hosts=$1 -P /ycsb/workloads/workloada
+if [ ! -z "$OPERATIONCOUNT" ]; then
+    OPERATIONCOUNT="-p operationcount=$OPERATIONCOUNT"
+fi
+
+/ycsb/bin/ycsb load cassandra-10 -p hosts=$1 -P /ycsb/workloads/workloada $RECORDCOUNT > /dev/null
```
Comment (neo-apz) [inline benchmarks/data-serving/client/docker-entrypoint.sh]: Have you tried this?

**Comment by ustiugov** [comment]
yes

File: benchmarks/data-serving/client/docker-entrypoint.sh (hunk 1 above)
Comment (neo-apz) [inline benchmarks/data-serving/client/docker-entrypoint.sh]: So as an example, what would be the value of $RECORDCOUNT ? 100000?

**Comment by ustiugov** [comment]
any positive integer.

**Pull request merged**